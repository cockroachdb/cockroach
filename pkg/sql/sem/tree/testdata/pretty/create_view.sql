CREATE VIEW startrek.quotes_per_season (season, quotes)
AS SELECT startrek.episodes.season, count(*)
FROM startrek.quotes
JOIN startrek.episodes
ON startrek.quotes.episode = startrek.episodes.id
GROUP BY startrek.episodes.season
