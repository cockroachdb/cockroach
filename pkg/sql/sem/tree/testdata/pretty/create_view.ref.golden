// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
CREATE VIEW startrek.quotes_per_season (
	season,
	quotes
)
	AS
		SELECT
			startrek.episodes.season,
			count(
				*
			)
		FROM
			startrek.quotes
			JOIN
				startrek.episodes
			ON
				startrek.quotes.episode
				= startrek.episodes.id
		GROUP BY
			startrek.episodes.season

20:
--------------------
CREATE VIEW startrek.quotes_per_season (
	season,
	quotes
)
	AS
		SELECT
			startrek.episodes.season,
			count(*)
		FROM
			startrek.quotes
			JOIN
				startrek.episodes
			ON
				startrek.quotes.episode
				= startrek.episodes.id
		GROUP BY
			startrek.episodes.season

34:
----------------------------------
CREATE VIEW startrek.quotes_per_season (
	season,
	quotes
)
	AS
		SELECT
			startrek.episodes.season,
			count(*)
		FROM
			startrek.quotes
			JOIN startrek.episodes
			ON
				startrek.quotes.episode
				= startrek.episodes.id
		GROUP BY
			startrek.episodes.season

46:
----------------------------------------------
CREATE VIEW startrek.quotes_per_season (
	season,
	quotes
)
	AS
		SELECT
			startrek.episodes.season, count(*)
		FROM
			startrek.quotes
			JOIN startrek.episodes
			ON
				startrek.quotes.episode
				= startrek.episodes.id
		GROUP BY
			startrek.episodes.season

55:
-------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes)
	AS
		SELECT
			startrek.episodes.season, count(*)
		FROM
			startrek.quotes
			JOIN startrek.episodes
			ON
				startrek.quotes.episode
				= startrek.episodes.id
		GROUP BY
			startrek.episodes.season

61:
-------------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes)
	AS
		SELECT
			startrek.episodes.season, count(*)
		FROM
			startrek.quotes
			JOIN startrek.episodes
			ON startrek.quotes.episode = startrek.episodes.id
		GROUP BY
			startrek.episodes.season

84:
------------------------------------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes)
	AS
		SELECT
			startrek.episodes.season, count(*)
		FROM
			startrek.quotes
			JOIN startrek.episodes ON startrek.quotes.episode = startrek.episodes.id
		GROUP BY
			startrek.episodes.season

100:
----------------------------------------------------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes)
	AS
		SELECT
			startrek.episodes.season, count(*)
		FROM
			startrek.quotes JOIN startrek.episodes ON startrek.quotes.episode = startrek.episodes.id
		GROUP BY
			startrek.episodes.season

176:
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes)
	AS SELECT startrek.episodes.season, count(*) FROM startrek.quotes JOIN startrek.episodes ON startrek.quotes.episode = startrek.episodes.id GROUP BY startrek.episodes.season

228:
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
CREATE VIEW startrek.quotes_per_season (season, quotes) AS SELECT startrek.episodes.season, count(*) FROM startrek.quotes JOIN startrek.episodes ON startrek.quotes.episode = startrek.episodes.id GROUP BY startrek.episodes.season


